apps/fz_http/lib/fz_http_web/mailer.ex

```diff
@@ -0,0 +1,43 @@
+defmodule FzHttpWeb.Mailer do
+  @moduledoc """
+  Outbound Email Sender.
+  """
+
+  use Swoosh.Mailer, otp_app: :fz_http
+
+  alias Swoosh.{Adapters, Email}
+
+  @provider_mapping %{
+    "smtp" => Adapters.SMTP,
+    "mailgun" => Adapters.Mailgun,
+    "mandrill" => Adapters.Mandrill,
+    "sendgrid" => Adapters.Sendgrid,
+    "post_mark" => Adapters.Postmark,
+    "sendmail" => Adapters.Sendmail
```

Comment:
Why do we limit ourselves to those providers while Swoosh supports much more of them? I guess we should accept those aliases but if the alias is now known it should be just an adapter name (module).

Reply:
Short answer: to savef 

Hm, so you're thinking the email configuration would look like this?

```
OUTBOUND_EMAIL_PROVIDER="Swoosh.Adapters.SMTP"
OUTBOUND_EMAIL_CONFIGS='{"Swoosh.Adapters.SMTP": { ... }, "Swoosh.Adapters.Sendgrid": {}}'
```

Perhaps this would be better suited to a UI (and REST API) configuration, or we sent more emails other than password reset / magic link. Only 2% of instances have outbound email configured.

Reply:
I was thinking:
```
OUTBOUND_EMAIL_PROVIDER_ADAPTER="Swoosh.Adapters.SMTP"
OUTBOUND_EMAIL_PROVIDER_ADAPTER_OPTS="{...}"
```

Reply:
Hmmm, yeah let me make a note of this of #1343 and maybe save this change for that work if you're alright with that. Otherwise we need to support both env var formats, which probably isn't too bad, but if we're changing these we probably want to add more supported fields, like `reply-to`, `name` in the from, etc.